provision/minikube/Taskfile.yaml

```diff
@@ -13,6 +13,7 @@ vars:
   KC_DB_POOL_MIN_SIZE: '{{default "5" .KC_DB_POOL_MIN_SIZE}}'
   KC_STORAGE: '{{default "" .KC_STORAGE}}'
   KC_DATABASE: '{{default "postgres" .KC_DATABASE}}'
+  KC_KEYCLOAK_IMAGE: '{{default "" .KC_KEYCLOAK_IMAGE}}'
```

Comment:
shouldn't this also include the changes for the resources we are pulling such as these, to match the version of the image artifact we are specifying in the KC_KEYCLOAK_IMAGE ? (Also feel like KC and KEYCLOAK are kind of redundant ain't it ?)

```
  - kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/keycloaks.k8s.keycloak.org-v1.yml
      - kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/keycloakrealmimports.k8s.keycloak.org-v1.yml
      - kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/kubernetes.yml || (kubectl -n keycloak delete deployment/keycloak-operator && kubectl -n keycloak apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/refs/tags/nightly/kubernetes/kubernetes.yml)
      
```

Reply:
I was thinking to add support for building a operator image locally eventually, and I plan to use "KC_OPERATOR_IMAGE` for that. Still not 100% convinced on the names, yet. 

The resources you're pointing to are the resources of the Keycloak operator -- those need to change only when we make the operator configurable.

Reply:
As discussed: KC_CONTAINER_IMAGE and KC_OPERATOR_IMAGE

Reply:
thanks. I think that works for me.